### DISCUSSION TITLE
What is the communication channel chosen by this developer staff to receive a request for a new localization activation?

### DISCUSSION THREAD

**Discussion by Ricky-Tigg** [dyne/tomb#514] [Q&A] [answered]
Good morning. Judging by
- the existence of the` /extras/translations` directory and its contents, support for localization exists.
- a ticket, a localization service platform has been chosen to host your localization project – Weblate.
- an investigation on the said platform, there is a project named _tomb_.

Finally. it therefore remains to be determined
- if the said project on weblate.org and the present one are linked.
- the preference of this developer staff as to the communication channel to use to send a request for a new localization activation on the said localization platform. I doubt that such a request is expected be made here, on this platform, which is not dedicated for this task.

**Comment by jaromil** [comment, marked as answer]
Hi!

Yes we use weblate and tomb there is connected to here.

You can just drop the request here or on our chat channels on dyne.org/linktree or privately by email to info@dyne.org

Cheers
